Decred Wallet/Features/More/More Features/Settings/SettingsController.swift

```diff
@@ -209,12 +251,56 @@ class SettingsController: UITableViewController  {
                     StartupPinOrPassword.set(sender: self, done: self.checkStartupSecurity)
                 }
                 
-            case 1: // change startup pin/password
+            case 1: // use biometric
+                self.promptForStartupPinOrPassword() { pinOrPassword, _, dialogDelegate in
+                    self.verifyPinOrPassword(startupPinOrPassword: pinOrPassword, dialogDelegate: dialogDelegate)
+                }
+                
+            case 2: // change startup pin/password
                 StartupPinOrPassword.change(sender: self, done: self.checkStartupSecurity)
                 
             default:
                 break
             }
         }
     }
+    
+    func promptForStartupPinOrPassword(callback: @escaping SecurityCodeRequestCallback) {
```

Comment:
This function name is generic, but the body isn't very generic as the prompt is fixed - "Enable with startup code". Modify this function signature to accept a prompt instead of building one here.
```suggestion
    func promptForStartupPinOrPassword(prompt: String, callback: @escaping SecurityCodeRequestCallback) {
```
_Alternatively,_
Can remove this function and move the code over to where the function is currently called. There isn't much code here and the function isn't called from more than 1 place.

Reply:
the function is used in the same file

Reply:
modified signature